R/score_transition_risk.R

```diff
@@ -92,6 +92,11 @@ score_transition_risk <-
     nest_levels(trs_product, trs_company)
   }
 
+#' Calulate `transition_risk_score` and `benchmark_tr_score` columns
+#'
+#' @param data Dataframe.
+#' @keywords internal
+#' @export
 create_tr_benchmarks_tr_score <- function(data) {
```

Comment:
tiltIndicatorAfter depends on tiltIndicator - not the other way around. 

If you now make tiltIndicator depend on tiltIndicator after, then we end up with a circular dependency grapgh. See "Problems" at https://en.wikipedia.org/wiki/Circular_dependency


But do think it would work in tiltIndicator "as is"? I see it uses column names that tiltIndicator doesn't know about. That means we'll need to re-write it -- or write an adaptor for it to work with the more general data structure that tiltIndicator manipulates. 

So you can leave this function alone here for now, and create a PR in tiltIndicator adding a compatible implementation there.

Later if you want you may export it from there and use it here (but we'll need an adaptor for the difference in columns names).

We may see this challenge more clearly once you try use it there. So you can try any idea and see how it goes. You can experiment using PR branches rather than merging to `main`.

Reply:
Makes sense! Other than different column names, the function will be used as is. I will copy the function to tiltIndicator and pass the column names as arguments so that the same function can then be exported to tiltIndicatorAfter, and we can replace the function from tiltIndicator.

Reply:
Sounds like a great idea, and it's an example of the flexibility we gain when we write code that is decoupled from implementation details (i.e. column names).